src/content/docs/mobile-monitoring/new-relic-monitoring-react-native/react-native-installation.mdx

```diff
@@ -0,0 +1,123 @@
+---
+title: Monitor your React Native application 
+tags:
+  - Mobile monitoring
+  - New Relic Mobile
+  - Configuration
+metaDescription: Instructions for installing and configuring the New Relic React Native agent.
+---
+
+import reactnativejavascripterrors from 'images/react-native-java-script-errors.png'
+
+With our React Native agent, your team can monitor the performance of its hybrid apps and identify code errors. Our agent collects crash data, network traffic, and other information for hybrid apps using native components. This allows your team to understand the health of your hybrid app regardless of the platform. We empower your team to make more informed development choices by providing insight into Javascript errors, distributed tracing, network instrumentation, and more. 
+
+## Install the React Native agent [#Install-the-React-Native-agent]
+
+
+### Requirements [#Requirements]
+
+Before you install the React Native agent, ensure your app meets these requirements:
+
+  * Android API 21 or higher
+  * iOS 10
+  * React Native version 0.63 or higher
+  * [iOS native requirements](/docs/mobile-monitoring/new-relic-mobile-ios/get-started/new-relic-ios-compatibility-requirements)
+  * [Android native requirements](/docs/mobile-monitoring/new-relic-mobile-android/get-started/new-relic-android-compatibility-requirements)
+
+Native support levels are based on [React Native requirements](https://github.com/facebook/react-native#-requirements).
```

Comment:
We could have this inside a callout. Maybe a tip or an important?

Reply:
Will do, thanks!